### PR TITLE
feat: Add `aws-ruby-step-functions-express` example

### PR DESCRIPTION
**Pull request by Pigius**
# Big picture behind

Currently, there aren't many examples of using the serverless approach with Ruby for AWS. Ruby is a great and mature language with a great community and lots of gems.

# General message

There are not many examples of Step Functions, and especially with the Express Workflow type. 

Luckily we also now have Workflow Studio for AWS Step Functions, so it's easier to design state machines.

This is an example of using `AWS Step Functions` `Express` Workflow Type. It uses `AWS Lambda`, `DynamoDB`, `Amazon SES`, `API Gateway` and `flows` from `Step Functions`. It's a simple workflow, it adds the user to the database (`DynamoDB`) and if the save is successful, we send an email notification (`AWS Lambda` + `Amazon SES`). We also have error handling.

Diagrams could be found here:
express-workflow-draw
stepfunctions_express_graph
express-diagram-visual

Extra points:
- Used the Ruby.27 runtime, as for the Ruby2.5 will be End of Support at the end of August.

### CODE REVIEW

File: aws-ruby-step-functions-express/README.md [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+
+Make sure to validate `sender` and `recipient` emails within the `Amazon SES` (most likely for testing it, it will be one `email`):
+
+![verify-email](./images/verify-email.png)
+
+Later on within `serverless.yml` edit in `custom` section, both `sender` and `recipient`.
+
+## Usage
+
+There are two possible of usage:
```
Comment (pgrzesik) [inline aws-ruby-step-functions-express/README.md:102]: Maybe rephrase this as `There are two possible ways of invoking the example` or something along those lines?

File: aws-ruby-step-functions-express/README.md [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+  "id": "The name of the employee",
+  "jobTitle": "the name of the employee's position"
+}
+```
+As a response you will get:
+
+```json
+{
+    "executionArn": "arn:aws:states:your-region:XXXXXXXXXXX:express:add-employee-and-send-email-state-machine-dev:e9fcce40-0642-48cb-bd9a-4d3d9cea7bcc:da5ee796-5944-4426-9ca2-410a5003ceee",
+    "startDate": 1.62885382751E9
```
Comment (pgrzesik) [inline aws-ruby-step-functions-express/README.md:121]: Nitpick: Let's change formatting of this value as I'm assuming it's timestamp, right?

**Review by pgrzesik — COMMENTED** [review]
Thank you @Pigius, great job as always :bow: I only have two nitpicks and we should be good to go :raised_hands:

File: aws-ruby-step-functions-express/README.md (hunk 1 above)
Comment (Pigius) [reply aws-ruby-step-functions-express/README.md]: Fixed!

File: aws-ruby-step-functions-express/README.md (hunk 2 above)
Comment (Pigius) [reply aws-ruby-step-functions-express/README.md:121]: It might be strange, but this is a response from the AWS Step Functions where you will trigger it: https://docs.aws.amazon.com/step-functions/latest/dg/tutorial-api-gateway.html

image

**Review by pgrzesik — APPROVED** [review]
Great job @Pigius :clap:
